src/components/CourseGradeItem.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
```

Comment:
Please help think of better name for CourseGradeItem and GradeItem. Both are GradeItem but are used in different pages.